components/common/NewTable.tsx

```diff
@@ -102,7 +108,7 @@ export default function NewTable({
 
   return (
     <OutlinedBox>
-      <Table variant="simple" fontSize="sm">
+      <Table variant="simple" fontSize="sm" size={size ? size : 'md'}>
```

Comment:
```suggestion
      <Table variant="simple" fontSize="sm" size={size ?? 'md'}>
```

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator meant for this use case.